src/main/java/com/example/Storehouse/service/ResponseService.java

```diff
@@ -0,0 +1,170 @@
+package com.example.Storehouse.service;
+
+import com.example.Storehouse.entity.Products;
+import com.example.Storehouse.entity.RefundProducts;
+import com.example.Storehouse.exception.exceptions.WrongQuantityException;
+import com.example.Storehouse.logger.Print;
+import com.example.Storehouse.models.product.Product;
+import com.example.Storehouse.models.product.RequiredProduct;
+import com.example.Storehouse.models.request.CheckQuantityOfProductRequest;
+import com.example.Storehouse.models.request.MultiProductRequest;
+import com.example.Storehouse.models.request.RefundRequest;
+import com.example.Storehouse.models.request.SingleProductRequest;
+import com.example.Storehouse.models.response.CheckQuantityResponse;
+import com.example.Storehouse.models.response.MultiProductsResponse;
+import com.example.Storehouse.models.response.RefundResponse;
+import com.example.Storehouse.models.response.SingleProductResponse;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class ResponseService implements iResponseService {
+    @Autowired
+    private Print print;
+    @Autowired
+    private  StorehouseService service;
+    private static final List<Product> productList = new ArrayList<>();
+
+    private void showInfoAboutRequest(Object obj) {
+        print.info("New request. " + obj.toString());
```

Comment:
all methods of Logger service should not be wrapped in another methods inside this or other class.
for example instead of

private void checkingTheProductInTheDatabase(SingleProductRequest singleProductRequest) {
        showInfoAboutRequest(singleProductRequest);
      ....
      ....
        requestProcessedSuccessfully();
    }

use:
private void checkingTheProductInTheDatabase(SingleProductRequest singleProductRequest) {
        loggerService.showInfoAboutRequest(singleProductRequest);
      ....
      ....
       loggerService.requestProcessedSuccessfully();
    }

this makes code easy to read, when i see loggerservice object i understand that this method only for logging and ignoring it. and focusing only on other methods.

Reply:
done